docs/services/kafka.md

```diff
@@ -55,8 +55,7 @@ $ ./kafka-topics.sh --alter --zookeeper=<zookeeper host>:2181 --topic <topci> --
 Pillowtop process configurations as described in the [CommCare docs](https://commcare-hq.readthedocs.io/pillows.html#parallel-processors).
 
 ### Move partitions
-**NOTE**: All processes accessing Kafka should be shutdown prior to following this process to avoid
- errors publishing or consuming Kafka messages.
+**NOTE**: This can be done while all services are online
```

Comment:
👍 

What changed that caused us to flip our position on this?

Reply:
I'm not sure why we had this initial position in the first place which is why I tagged Simon. The way kafka does this by default is quite safe. The (automated) process was:

1. Add partition to new broker as replica
2. Replicate the full partition data to the new broker
3. Elect the new broker as the leader
4. Remove the partition from the old broker

So its quite safe. I could possibly see some issues if your kafka cluster is totally maxed out on resources and you tried to do it at peak loads (later versions add some network throttling), but in general kafka is super light on resources and this is a safe process